### DISCUSSION TITLE
refund

### DISCUSSION THREAD

**Discussion by Biantuadi** [community/community#39185] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

Good morning,
Last week I had subscribed for github copilot 10€ per me, and it was last week that I activated github for students and I just learned that copyilot is free for students but yet I was prelifted 10€ today around 8am.
I tried to stop the subscription for me everything was okey; but yet I just received an email with a fee of 80€.... 
I don’t understand why.
thank you.

**Comment by david-engelmann** [comment, marked as answer]
[You can find the support section for Github for Student here](https://support.github.com/). If you reach out to support@github.com, they can also resolve your issue!

**Reply by Biantuadi** [reply]
Thanks;

**Reply by david-engelmann** [reply]
make sure to mark it as answered!

**Reply by nonoBore** [reply]
Associates in 18 years … oh yes an no school Nuckle head

**Reply by nonoBore** [reply]
Electic or hydo
